examples/buildkit0/buildkit.go

```diff
@@ -17,8 +17,8 @@ type buildOpt struct {
 func main() {
 	var opt buildOpt
 	flag.BoolVar(&opt.withContainerd, "with-containerd", true, "enable containerd worker")
-	flag.StringVar(&opt.containerd, "containerd", "v1.0.3", "containerd version")
-	flag.StringVar(&opt.runc, "runc", "9f9c96235cc97674e935002fc3d78361b696a69e", "runc version")
+	flag.StringVar(&opt.containerd, "containerd", "v1.1.0", "containerd version")
+	flag.StringVar(&opt.runc, "runc", "69663f0bd4b60df09991c08812a60108003fa340", "runc version")
```

Comment:
This is the version defined in containerd/vendor.conf, which is actually used in containerd's CI.

Reply:
Shouldn't we leave this and CI to the version in https://github.com/containerd/containerd/blob/v1.1.0/RUNC.md ? In vendor we can use a different version. Do you know why they differ in containerd?

Reply:
I guess they just differ accidentally, and the revision specified in vendor.conf is the desired one. https://github.com/containerd/containerd/pull/2337